client/templates/hangout/hangout-clone.js

```diff
@@ -53,6 +53,10 @@ Template.cloneHangoutModal.onRendered(function() {
         data: studyGroups
       });
 
+      const groupId = templateInstance.data.hangout.data.group.id || '';
+      if (groupId !== '') {
+    -   instance.$(".study-group-single").val(groupId).trigger('change');
```

Comment:
This is awesome, @railsstudent. Thanks for addressing this issue.

Question: Was this `-` on line 58 added by mistake, or am I missing something?

Reply:
It is a typo. Added by mistake.

Reply:
I will make the correction tonight

Reply:
@lpatmo  Fixed. Please kindly review